src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentCostPerDay = 40;
+  const threeDaysDiscount = 20;
+  const weeklyDiscount = 50;
```

Comment:
do not use numbers in names - what if we change 3 to 4?
You will change the variables names then?